src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/parameterassignment/InputParameterAssignmentWithUnchecked.java

```diff
@@ -29,6 +29,11 @@ public void test(String hidden) {
         hidden += "test";
     }
 
+    // parameter name must be the same token name
+    void foo5(int EXPR) {
```

Comment:
http://rveach.no-ip.org/checkstyle/regression/pitest-reports/37/pitest-coding/com.puppycrawl.tools.checkstyle.checks.coding/ParameterAssignmentCheck.java.html#org.pitest.mutationtest.report.html.SourceFile@25925217_197
The only way to kill this check was to have a non-ident token have the same name as the parameter. I am sure regression would not have found anything like this since it has to match our internal structure.